cvr_osc_lib/osc_interface.py

```diff
@@ -28,30 +31,47 @@
 
 
 class EndpointPrefix(str, Enum):
-    avatar_change = '/avatar/change'
-    avatar_parameter = '/avatar/parameter'
-    avatar_parameters_legacy = '/avatar/parameters/'
-    input = '/input/'
-    prop_create = '/prop/create'
-    prop_delete = '/prop/delete'
-    prop_available = '/prop/available'
-    prop_parameter = '/prop/parameter'
-    prop_location = '/prop/location'
-    prop_location_sub = '/prop/location_sub'
-    tracking_device_status = '/tracking/device/status'
-    tracking_device_data = '/tracking/device/data'
-    tracking_play_space_data = '/tracking/play_space/data'
-    config_reset = '/config/reset'
-
-
-# def osc_default_handler(address: str, *args) -> None:
+    """ The OSC endpoint prefixes """
+    AVATAR_CHANGE = '/avatar/change'
+    AVATAR_PARAMETER = '/avatar/parameter'
+    AVATAR_PARAMETERS_LEGACY = '/avatar/parameters/'
+    INPUT = '/input/'
+    PROP_CREATE = '/prop/create'
+    PROP_DELETE = '/prop/delete'
+    PROP_AVAILABLE = '/prop/available'
+    PROP_PARAMETER = '/prop/parameter'
+    PROP_LOCATION = '/prop/location'
+    PROP_LOCATION_SUB = '/prop/location_sub'
+    TRACKING_DEVICE_STATUS = '/tracking/device/status'
+    TRACKING_DEVICE_DATA = '/tracking/device/data'
+    TRACKING_PLAY_SPACE_DATA = '/tracking/play_space/data'
+    CONFIG_RESET = '/config/reset'
```

Comment:
I'm not using pylint, this project is using `flake8`
It allows Lower Case

Reply:
fixed in [a53fc86](https://github.com/kafeijao/cvr_osc_lib_py/pull/3/commits/a53fc8645cb1c7521b50d49f3bb50eca83e0f015)